torch/_functorch/aot_autograd.py

```diff
@@ -1324,9 +1327,7 @@ def aot_dispatch_base(flat_fn, flat_args: List[Tensor], aot_config: AOTConfig):
         fw_module.graph.eliminate_dead_code()
         fw_module.recompile()
 
-    if config.debug_graphs:
-        log.debug(f"====== Forward (only) graph {aot_config.aot_id} ======")
-        log.debug(fw_module.print_readable(print_output=False))
+    log.debug(AOTForwardGraphLogRec(f"====== Forward (only) graph {aot_config.aot_id} ======\n", fw_module))
```

Comment:
Shouldn't the AOTForwardGraphLogRec take care of formatting the header line here?